com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -626,6 +654,154 @@ static void OnLightingDataCleared()
             Clear();
         }
 
+        static void GetShaderCoeffsL0L1(ref SphericalHarmonicsL2 sh, NativeArray<Vector4> target, int offset)
+        {
+            target[offset + 0] = new Vector4(sh[0, 0], sh[1, 0], sh[2, 0], sh[0, 1]);
+            target[offset + 1] = new Vector4(sh[1, 1], sh[1, 2], sh[1, 3], sh[0, 2]);
+            target[offset + 2] = new Vector4(sh[2, 1], sh[2, 2], sh[2, 3], sh[0, 3]);
+        }
+
+        static void GetShaderCoeffsL2(ref SphericalHarmonicsL2 sh, NativeArray<Vector4> target, int offset)
+        {
+            target[offset + 0] = new Vector4(sh[0, 4], sh[0, 5], sh[0, 6], sh[0, 7]);
+            target[offset + 1] = new Vector4(sh[1, 4], sh[1, 5], sh[1, 6], sh[1, 7]);
+            target[offset + 2] = new Vector4(sh[2, 4], sh[2, 5], sh[2, 6], sh[2, 7]);
+            target[offset + 3] = new Vector4(sh[0, 8], sh[1, 8], sh[2, 8], 1f);
+        }
+
+        static void GetBlobFileNames(string sourceFilename, out string cellDataFilename, out string cellSupportDataFilename )
+        {
+            cellDataFilename = System.IO.Path.ChangeExtension(sourceFilename, "CellData.bytes");
+            cellSupportDataFilename = System.IO.Path.ChangeExtension(sourceFilename, "CellSupportData.bytes");
+        }
+
+        static void WriteBakingCells(ProbeVolumeAsset asset, List<BakingCell> bakingCells)
```

Comment:
Flatten all cells into two byte streams: one for runtime data and one for debug data.

Reply:
Now there are three byte streams (the essential data, the optional data - aka L2, and the support data - aka debug) as discussed.